content/getting-started/cf-ssh.md

```diff
@@ -21,7 +21,8 @@ Our `cf-ssh` is customized to our Cloud Foundry installation so please **do not
 
 ### Usage
 
-In your project folder run `cf-ssh APP-NAME` replacing `APP-NAME` with the app name.
-Currently, `cf-ssh` requires a `manifest.yml` file. Please create one before running it. You can also use the `-f` flag to specify a manifest file.
+1. If you don't have an [application manifest](http://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) yet, generate one from an existing app with `cf create-app-manifest <APP_NAME>`.
+1. In your project folder, run `cf-ssh <APP_NAME> -f <PATH_TO_MANIFEST>`.
```

Comment:
Maybe we can add a note that `-f` is only required if you don't use the name `manifest.yml`?